### ISSUE TITLE
Re-test with missing times/best segments/etc

### ISSUE THREAD

**Issue by Komarulon** [Komarulon/LiveSplit.GradedSplits#20]
From conversation with Azder:

i think i found a bug. if you load new splits that don't have any best segments, it basically acts like it removes the whole splits component from your layout

and even if you try to re-add the graded splits, or even just a regular splits component, it still refuses to show any of them

that bug seems to be saved in the layout, because i switched to other layouts and those worked fine

i tried remaking the layout, that worked
but then i loaded said brand new splits file and it immediately broke again
and stayed broken even when i switched to splits that had data, even when i reloaded livesplit with them
so basically i have to avoid loading files that don't have best segments, if the icon override feature is turned on
if the icon override is off then it seems to work fine

it might be if split times are missing, too. not sure
